logging/google/cloud/logging/entries.py

```diff
@@ -74,11 +74,16 @@ class _BaseEntry(object):
     :param http_request: (optional) info about HTTP request associated with
                          the entry.
 
+    :type trace: str
+    :param trace: (optional) traceid to apply to the entry.
+
+
```

Comment:
Please move this below `resource, to match the order in the actual function definition.